src/snadra/__main__.py

```diff
@@ -11,9 +11,12 @@
 """
 import asyncio
 
+from snadra._config import setupConfig
 from snadra._core.parsers import CommandParser
 
 if __name__ == "__main__":
+    setupConfig()
```

Comment:
```suggestion
    setup_config()
```

We are not coding in java:)